docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md

```diff
@@ -717,11 +717,109 @@ target:
 
 Refer to the [live-migration-with-fall-back.yaml](https://github.com/yugabyte/yb-voyager/blob/{{< yb-voyager-release >}}/yb-voyager/config-templates/live-migration-with-fall-back.yaml) template for more information on the available global, source, and target configuration parameters supported by Voyager.
```

Comment:
Above it says "If you want to assess your migration, create the database with colocation"
This is misleading. The one does not depend on the other.
"If you intend to perform a migration assessment, note that the assessment provides recommendations on which tables in the source database to colocate. To ensure that you will be able to colocate tables, create your target database with colocation set to TRUE using the following command:"